addons/website_slides_survey/models/slide_channel.py

```diff
@@ -1,16 +1,38 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import fields, models, api
 
 
 class Channel(models.Model):
     _inherit = 'slide.channel'
 
     nbr_certification = fields.Integer("Number of Certifications", compute='_compute_slides_statistics', store=True)
+    certified_partners_count = fields.Integer('Number of certified partners', compute="_compute_certified_partners_count")
```

Comment:
Same here, I already have the same kind of computation. Somehow we will have to converge. Someday.